frigate/api/event.py

```diff
@@ -302,8 +302,21 @@ def events_explore():
         .dicts()
     )
 
-    events = query.iterator()
-    return jsonify(list(events))
+    events = list(query.iterator())
```

Comment:
I think we should make this conditional with a query arg or something, that way we don't incur an api slowdown if we don't want this information

Reply:
Not quite sure what you mean. This is for the `/events/explore` endpoint, and there's already a limit query arg. We use all of the fields that are returned for the details pane.

Reply:
Oh nevermind, thought this was /events